zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/DeploymentTransformer.java

```diff
@@ -36,11 +36,10 @@ public final class DeploymentTransformer {
 
   private static final Logger LOG = Loggers.PROCESS_PROCESSOR_LOGGER;
 
-  private static final DeploymentResourceTransformer UNKNOWN_RESOURCE =
-      new UnknownResourceTransformer();
-
```

Comment:
❌ Let's restore this, we still haven't developed any concepts for a general-resource transformation mechanism.